src/api/app/views/webui/users/notifications/_notification_action_bar.html.haml

```diff
@@ -6,5 +6,7 @@
           = check_box_tag('select-all-notifications', 1, false, class: 'custom-control-input')
           %label.custom-control-label.align-middle.mr-4#select-all-label{ for: 'select-all-notifications' } Select All
           - button_text = type == 'read' ? "Mark as 'Unread'" : "Mark as 'Read'"
-          = button_tag(type: 'submit', class: 'btn btn-sm btn-outline-success px-3 disabled', id: 'done-button') do
+          = button_tag(type: 'submit', class: 'btn btn-sm btn-outline-success px-3', id: 'done-button', disabled: 'disabled',
+                       'data-disable-with': "<i class='fas fa-spinner fa-spin'></i>".html_safe) do
```

Comment:
It looks a bit weird with only a spinner, but it's not a blocker.

Reply:
@dmarcoux I agree, the spinner does now only show up on the button that got clicked